lib/rollout.rb

```diff
@@ -88,7 +88,7 @@ def id_user_by
       end
 
       def user_in_percentage?(user)
-        Zlib.crc32(user_id_for_percentage(user)) % 100_000 < @percentage * 1_000
+        Zlib.crc32(user_id_for_percentage(user)) < (2**32 - 1) / 100.0 * @percentage
```

Comment:
Can you please extract `(2**32 - 1) / 100.0` into the constant? It'd be very inefficient to calculate this value on each request.